evidence-based-complementary-and-alternative-medicine.csl

```diff
@@ -0,0 +1,313 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Evidence-Based Complementary and Alternative Medicine</title>
+    <title-short>eCAM</title-short>
+    <id>http://www.zotero.org/styles/evidence-based-complementary-and-alternative-medicine</id>
+    <link href="http://www.zotero.org/styles/evidence-based-complementary-and-alternative-medicine" rel="self"/>
+    <link href="http://www.hindawi.com/journals/ecam/guidelines/" rel="documentation"/>
+    <author>
+      <name>Seungwon Shin</name>
+      <email>justin0624@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field=" medicine"/>
+    <category field=" biology"/>
```

Comment:
Can you remove the spaces here? " medicine" and " biology" should be "medicine" and "biology", respectively.